conf/machine/include/imx-base.inc

```diff
@@ -69,11 +69,11 @@ SPL_BINARY ??= "${@get_spl_binary(d)}"
 
 IMX_DEFAULT_MFGTOOL = "${@bb.utils.contains('IMX_DEFAULT_BOOTLOADER', 'u-boot-imx','u-boot-imx-mfgtool', 'u-boot-fslc', d)}"
 
-PREFERRED_PROVIDER_u-boot ??= "${IMX_DEFAULT_BOOTLOADER}"
-PREFERRED_PROVIDER_u-boot-mfgtool ??= "${IMX_DEFAULT_MFGTOOL}"
-PREFERRED_PROVIDER_virtual/bootloader ??= "${IMX_DEFAULT_BOOTLOADER}"
+PREFERRED_PROVIDER:u-boot ??= "${IMX_DEFAULT_BOOTLOADER}"
```

Comment:
Are you sure @thochstein ? I think that we need to keep the original syntax -> `PREFERRED_PROVIDER_` here

Reply:
Yes, I believe you're right. Will fix.

Reply:
done, I have squashed this one into original commit from pull 1